dockerfiles/ci/buster/Dockerfile

```diff
@@ -237,9 +248,9 @@ RUN set -eux; \
     chown -R circleci:circleci /opt;
 
 # rust sha256sum generated locally after verifying it with sha256
-ARG RUST_VERSION="1.64.0"
-ARG RUST_SHA256_ARM="7d8860572431bd4ee1b9cd0cd77cf7ff29fdd5b91ed7c92a820f872de6ced558"
-ARG RUST_SHA256_X86="a893977f238291370ab96726a74b6b9ae854dc75fbf5730954d901a93843bf9b"
+ARG RUST_VERSION="1.71.0"
```

Comment:
On August 3rd, version 1.71.1 was released. I think we should probably ship that, which unfortunately means quite a bit of rebuilding 😅

Reply:
Here we go again ;-)

Reply:
done